src/ghosts.client.linux/nlog.config

```diff
@@ -6,7 +6,7 @@
       internalLogLevel="Error"
 >
   <targets>
-    <target name="logfile" xsi:type="File" fileName="logs/app.log" layout="${Date:universalTime=true}|${callsite}|${message}" archiveAboveSize ="1000000" maxArchiveFiles="2" />
+    <target name="logfile" xsi:type="File" fileName="logs/app.log" layout="${date:universalTime=true:format=dd/MM/yyyy H\:mm\:ss tt}|${callsite}|${message}" archiveAboveSize ="1000000" maxArchiveFiles="2" />
```

Comment:
Should it be `HH\:mm\:ss` instead of single `H` ?

Reply:
The other log does not have leading zeros, and the want was for them to match.